libs/core/src/lib/services/firestore/firestore-transaction/firestore-transaction.service.ts

```diff
@@ -58,7 +65,7 @@ export class FirestoreTransactionService {
               id: docId,
               timestamp,
               amount: amount ?? 0,

```

Comment:
Check if nullish coalescing can be removed

Reply:
Fixed in https://github.com/usamazansari/expenses-tracker/pull/74/commits/442489534fc8685d6df86656893e9bb0d8c81613